src/modules/users/add-user/add-user.html

```diff
@@ -57,7 +57,7 @@
             <div class="form-group required">
                 <label class="col-sm-2 col-md-1 control-label cp-padding" for="password">Confirm Password</label>
                 <div class="col-sm-8 col-md-6 inner-addon">
-                    <input type="{{ addUserCntrl.confirmPassword ? 'text' : 'password' }}" data-id="password" aria-describedby="passwordStatus" name="confirmPassword" ng-model="addUserCntrl.user.confirmPassword" minlength="8" required/>
+                    <input type="{{ addUserCntrl.confirmPassword ? 'text' : 'password' }}" data-id="password" aria-describedby="passwordStatus" name="confirmPassword" ng-model="addUserCntrl.user.confirmPassword" minlength="9" ng-maxlength="128" required/>
```

Comment:
if you have used ng-maxlength then why you have not used ng-minlength?

Reply:
Same as above comment.